pkgs/servers/home-automation/evcc/default.nix

```diff
@@ -0,0 +1,141 @@
+{ pkgs
+, lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, mockgen
+, nixosTests
+
+# updateScript
+, coreutils
+, curl
+, jq
+, git
+, gnused
+, nix-prefetch-git
+, nix-update
+, nodePackages
+}:
+
+let
+  nodejs = pkgs.nodejs-14_x;
+  nodeEnv = import ../../../development/node-packages/node-env.nix {
+    inherit (pkgs) stdenv lib python2 runCommand writeTextFile writeShellScript;
+    inherit pkgs nodejs;
+    libtool = if stdenv.isDarwin then pkgs.darwin.cctools else null;
+  };
+  evccNodePackages = import ./node-packages.nix {
+    inherit (pkgs) fetchurl nix-gitignore stdenv lib fetchgit;
+    inherit nodeEnv;
+  };
+
+  srcJSON = lib.importJSON ./src.json;
+  src = fetchFromGitHub {
+    owner = "andig";
+    repo = "evcc";
+    inherit (srcJSON) rev sha256;
+  };
+
+  nodeDependencies = (evccNodePackages.shell.override (old: {
+    inherit src;
+  })).nodeDependencies;
+in
+buildGoModule rec {
+  pname = "evcc";
+  version = "";
```

Comment:
How did that happen?

Reply:
Funny. Ran the update script, ran the tests, did the commit and didn't look at the changes.

Reply:
Can't comment on the line but ``src = ../../../../../../../../../tmp/tmp.TDkejGx5wj;`` in nodePackages also looks weird.